### DISCUSSION TITLE
Lancement du prototype!

### DISCUSSION THREAD

**Discussion by remillc-udem** [bibudem/coquille-web#1] [Announcements]
Salut!

Une version initiale entièrement fonctionnelle du prototype vient d'être versée dans le dépôt GitHub.

Le site est basé sur un template de site axé sur l'accessibilité et qui utilise Tailwind CSS.

Pour les techies, vous pouvez instancier un front end en moins de 5 minutes après avoir clôné le dépôt et avoir lu les instructions dans le fichier README.md.

Si l'envie vous prends de faire des commits, s.v.p. limitez vous à des changements apportés aux pages et leurs contenus uniquement pour l'instant. On prévoit adapter le layout général pour lui donner d'avantage de ressemblance avec un site de l'UdeM ainsi que de remplacer les pages et leurs contenus par quelque chose qui ressemble à une section typique de notre site.

Bonne exploration!

**Comment by msevigny** [comment]
Ça fonctionne du premier coup pour moi, en suivant les quelques étapes du Readme. Merci!
